EventView-1/src/main/java/com/eventview/repo/EventRepoImpl.java

```diff
@@ -23,32 +24,47 @@
 
     @Override
     public List<EventsPayload> getAllEvens() {
-        String esql = "select e.event_id,concat(u.first_name,' ',u.last_name) as full_name, et.event_type,e.event_date from events e left join users u on e.user_id = u.user_id left join eventtypes et on et.event_type_id=e.event_type_id";
-        return jdbcTemplate.query(esql, new EventsPayloadRowMapper());
+        String sql1 = "select e.event_id,concat(u.first_name,' ',u.last_name) as full_name, et.event_type,e.event_date from events e left join users u on e.user_id = u.user_id left join eventtypes et on et.event_type_id=e.event_type_id";
```

Comment:
told multiple times avoid concat here

Reply:
fixed except concat
will find a way and fix it